app/pages/TrustURNPage.scala

```diff
@@ -18,7 +18,7 @@ package pages
 
 import play.api.libs.json.JsPath
 
-case object TrustURNPage extends QuestionPage[String] {
+case object TrustURNPage extends QuestionPage[String] { // TODO: use a TRN class that extends TaxIdentificationNumber
```

Comment:
Is any refactoring to cover TODOs we can do part of this ticket? or create another ticket at-least and refer the number here

Reply:
pretty big already - i think a ticket would be better